frontend-react/package.json

```diff
@@ -81,8 +81,7 @@
         "yarn:show-outdated-packages": "yarn outdated",
         "run-build-dir": "yarn build:localdev:csp && yarn global add serve && serve -s build",
         "storybook": "npm-run-all -p watch-scss storybook:start",
-        "storybook:start": "start-storybook -p 6006",
```

Comment:
Removing this makes `npm run storybook` error out as it relies on `storybook:start` @stephenkao

Reply:
Whoops, will put up a fix!

Reply:
https://github.com/CDCgov/prime-reportstream/pull/9236